go-controller/pkg/node/node.go

```diff
@@ -138,9 +138,16 @@ func isOVNControllerReady(name string) (bool, error) {
 	err = wait.PollImmediate(500*time.Millisecond, 60*time.Second, func() (bool, error) {
 		stdout, _, err := util.RunOVSOfctl("dump-aggregate", "br-int")
 		if err != nil {
+			klog.Errorf("Error dumping aggregate flows: %v "+
```

Comment:
@abhat not sure this hsould be Errorf... we'll fail here if br-int is created yet (which ovn-controller does) but what if we get here before the ovn-controller pod has started? That's likely to happen at startup

Reply:
that is just a log, I can log it at Info level too. we are waiting for the 60 seconds for the ovn-controller to be up.